### PR TITLE
Add documentation on known bugs

### PR DESCRIPTION
**Pull request by ibusko**

### CODE REVIEW

**Comment by javerbukh** [comment]
I would also mention the data flashing bug, where if data addition/removal is done too quickly, the front-end can get "desynced" from the kernel. The solution for this is to just restart the kernel or restart jupyter notebook entirely if the bug persists.

File: docs/known_bugs.rst [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+second conversion is applied. It doesn't happen when no subset is selected
+and successive conversions are applied.
+
+
+Ghost subsets and models in spectrum viewer
+-------------------------------------------
+
+Mysterious extra subsets and models appear in the Specviz spectrum viewer
+after fitting a model. Pan/zoom and resize were also used, but it is unclear
+if they play a role or not in this bug:
```
Comment (PatrickOgle) [inline docs/known_bugs.rst:28]:
```suggestion
if they play a role or not in this bug.
```

File: docs/known_bugs.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+the colormap name is printed in the cell output.
+
+
+Cubeviz cube viewer contrast changes when collapsing jupyter scroll window
+---------------------------------------------------------------------------
+
+In order to see the full Cubeviz app in a Jupyter notebook, one can click on
+the side of the cell output to collapse or expand the scrollable window. This
+has the unintended consequence of changing the contrast of the image displayed
+in the Cubeviz cube viewer:
```
Comment (PatrickOgle) [inline docs/known_bugs.rst:61]:
```suggestion
in the Cubeviz cube viewer.
```

**Review by PatrickOgle — APPROVED** [review]
Looks good.  I didn't check the jdaviz issues list to see if this documentation covers them 
all, but the descriptions of the bugs here seem accurate to me, and follow the github issue.
I corrected two typos and committed the corrections.

**Comment by javerbukh** [comment]
Couple things:
1) Can you rebase from master to see if that fixes the CI issues?
2) Can you also add to the page the link to jdaviz issues in case a user finds a new bug? I'm not sure if the top or bottom of the page makes more sense. https://github.com/spacetelescope/jdaviz/issues/new

**Comment by ibusko** [comment]
@javerbukh I created a completely new PR (#293 ) from a fresh checkout of master to which I just added the new `known_bugs.rst` file and the modification of `index.rst`. That seems to have fixed the problem. But now we see new build errors, one of them related to a missing `docs/quickstart.rst` file.

**Comment by javerbukh** [comment]
This can be closed now that #293 is merged.

**Pull request closed without merging**